src/pages/order/OrderPage.jsx

```
@@ -0,0 +1,43 @@
+import OrderComponent from '../../components/OrderComponent';
+import { CheckOutComponent } from '../../components/TossCheckOutComponent';
+import FixedLayout from "../../components/FixedLayout.jsx";
+
+const OrderPage = () => {
+  const location = useLocation();
+  const initialOrderData = location.state?.orderData || null;
+
+  const [orderCompleteData, setOrderCompleteData] = useState(null);
+
+  useEffect(() => {
+    if (initialOrderData && initialOrderData.directCheckout) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)